recipes/src/main/resources/quarkus-updates/core/3alpha.yaml

```diff
@@ -1057,6 +1057,10 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: io.quarkus.arc.Priority
       newFullyQualifiedTypeName: jakarta.annotation.Priority
+  - org.openrewrite.text.FindAndReplace:
+      find: quarkus-bootstrap-maven-plugin
+      replace: quarkus-extension-maven-plugin
+      fileMatcher: '**/pom.xml'
```

Comment:
I'm not exactly happy we do it this way but I don't see a better option for now. It would probably be interesting to contribute it to OpenRewrite.

Reply:
Yeah, I asked in their Slack channel. I'll update it when I have a better solution

Reply:
Issue created: https://github.com/openrewrite/rewrite/issues/3047

Reply:
And PR ready: https://github.com/openrewrite/rewrite/pull/3048

Reply:
+1 on doing this with something better than text replace. PR on openrewrite looks good!